src/pages/variantEntity/TabSummary.tsx

```diff
@@ -1,49 +1,216 @@
+      ],
+      ['cadd', consequence.node.predictions?.cadd_rankscore],
+      ['dann', consequence.node.predictions?.dann_rankscore],
+      [
+        'lrt',
+        consequence.node.predictions?.lrt_pred,
+        consequence.node.predictions?.lrt_converted_rankscore,
+      ],
+      ['revel', consequence.node.predictions?.revel_rankscore],
+    ],
+    conservations: consequence.node.conservations?.phylo_p17way_primate_rankscore,
+    transcript: consequence.node.ensembl_transcript_id,
+  }));
+
+const TabSummary = ({ variantId }: OwnProps) => {
+  const { loading, data: rawData, error } = useTabSummaryData(variantId);
+
+  if (error) {
+    return (
+      <Result
+        status="500"
+        title="Server Error"
+        subTitle="An error has occured and we are not able to load content at this time."
+      />
+    );
+  }
+
+  const data = rawData as VariantEntity | undefined;
+
+  const consequences = (data?.consequences?.hits?.edges || []) as Consequence[];
 
   return (
     <Spin spinning={loading}>
-      <div className={'content-container'}>
-        <Card>
-          <div className={'grid-top-container'}>
-            <div className={'grid-top-container-item'}>
-              <div style={{ display: 'flex' }}>
-                <div style={{ width: '25%' }}>Chr</div>
-                <div style={{ width: '75%' }}>missing</div>
-              </div>
-              <div style={{ display: 'flex' }}>
-                <div style={{ width: '25%' }}>Start</div>
-                <div style={{ width: '75%' }}>33</div>
-              </div>
-              <div style={{ display: 'flex' }}>
-                <div style={{ width: '25%' }}>Allele Alt.</div>
-                <div style={{ width: '75%' }}>33</div>
-              </div>
-              <div style={{ display: 'flex' }}>
-                <div style={{ width: '25%' }}>Allele Réf.</div>
-                <div style={{ width: '75%' }}>33</div>
-              </div>
-            </div>
-            <div className={'grid-top-container-item'}>
-              <div className={'grid-top-container-item'}>
-                <div style={{ display: 'flex' }}>
-                  <div style={{ width: '25%' }}>rre</div>
-                  <div style={{ width: '75%' }}>33</div>
-                </div>
-              </div>
-            </div>
-          </div>
-        </Card>
-      </div>
+      <StackLayout vertical fitContent>
+        <Space direction={'vertical'} size={'large'}>
+          <Summary variant={data} />
+          {makeTables(consequences).map((tableData: TableGroup, index: number) => {
+            const symbol = tableData.symbol;
+            const omim = tableData.omim;
+            const biotype = tableData.biotype;
+            const orderedConsequences = tableData.consequences;
+            return (
+              <Table
+                key={index}
+                title={() => [symbol, omim, biotype].join(' ')}
+                bordered
```

Comment:
no bordered table.

We need to wrap it in a container as in Figma

Reply:
no, one return type